DiscordEnvironmentInfo/index.js

```diff
@@ -0,0 +1,39 @@
+const Plugin = module.parent.require('../Structures/Plugin');
+class DiscordEnvironmentInfo extends Plugin {
+    load() {
+        this.registerCommand({
+            name: 'envinfo',
+            info: 'Shows information about the current environment.',
+            func: this.envinfo.bind(this, true)
+        });
+    }
+    envinfo(args) {
+        const v = new Date().getTime();
+        webpackJsonp([], {[v]: (a, b, d) => {
```

Comment:
get the release channel faster: `Object.keys(process.versions).find(k => k.toLowerCase().startsWith("discord"))`

Discord for stable
DiscordPTB for ptb
DiscordCanary for canary
DiscordDevelopment for dev